security/cert_manager_operator/cert-manager-log-levels.adoc

```diff
@@ -0,0 +1,18 @@
+:_content-type: ASSEMBLY
+[id="cert-manager-log-levels"]
+= Configuring log levels for cert-manager and the {cert-manager-operator}
+include::_attributes/common-attributes.adoc[]
+:context: cert-manager-log-levels
+
+toc::[]
+
+To troubleshoot some issues with the cert-manager components and the {cert-manager-operator}, you can configure the log level verbosity.
```

Comment:
is `some` necessary? If yes, can `some` be defined here?

Reply:
`some` can be avoided